tests/pytests/test_multibyte_char_terms.py

```diff
@@ -482,77 +482,79 @@ def testDiacriticLimitation(env):
 
 @skip(cluster=True)
 def testStopWords(env):
-    '''Test that stopwords are not indexed, but for multibyte characters they
-    are not converted to lowercase correctly. This is a limitation that will be
-    fixed by MOD-8443'''
+    '''Test that stopwords using multibyte characters are converted to lowercase
+    correctly
+    '''
 
     conn = getConnectionByEnv(env)
-    # test with russian lowercase stopwords
+    # test with multi-byte lowercase stopwords
     env.cmd('FT.CREATE', 'idx1', 'ON', 'HASH', 'LANGUAGE', 'RUSSIAN',
-            'STOPWORDS', 3, 'и', 'не', 'от',
+            'STOPWORDS', 4, 'и', 'не', 'от', 'fußball',
             'SCHEMA', 't', 'TEXT', 'NOSTEM')
 
-    conn.execute_command('HSET', 'doc:1', 't', 'не ясно') # 1 term
+    conn.execute_command('HSET', 'doc:1', 't', 'не ясно fußball') # 1 term
     conn.execute_command('HSET', 'doc:2', 't', 'Мужчины и женщины') # 2 terms
     conn.execute_command('HSET', 'doc:3', 't', 'от одного до десяти') # 3 terms
     # create the same text with different case
-    conn.execute_command('HSET', 'doc:4', 't', 'НЕ ЯСНО')
+    conn.execute_command('HSET', 'doc:4', 't', 'НЕ ЯСНО FUßBALL')
```

Comment:
Anything special about `FUßBALL`?
Why was it added?

Reply:
The `ß` is special, it's folded to `ss`. At the beginning we were normalizing usint `tofold()`, now we are using `tolower()`.
But I don't see problem if we continue testing with `ß`